R/createInputForDescriptives.r

```diff
@@ -19,9 +19,11 @@
 #'@param impExpr Regular expression to identify imputed variables from variable labels in GADSdat
 #'object (Note: for multiple expressions, i.e. if \code{impExpr} is a character vector of length > 1,
 #'at least one expression should match to identify the variable as an imputed variable)
-#'@param scaleExpr Regular expression to identify scale variables from variable labels in GADSdat
+#'@param scaleExpr Regular expression to identify scale or fake scale variables from variable labels in GADSdat
 #'object (Note: for multiple expressions, i.e. if \code{scaleExpr} is a character vector of length > 1,
 #'at least one expression should match to identify the variable as a scale variable)
+#'@param itemExpr Regular expression to identify items which constitute a true scale.

```

Comment:
Makes sense to me to have these as explicit arguments! However, could the argument description mention explicitly that this refers to variable labels and not variable names?

Reply:
done